tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -142,6 +142,20 @@
      'target': 'mozilla-central-firefox-25.0a1.en-US.linux-i686.txt',
      'target_url': 'firefox/tinderbox-builds/mozilla-central-linux/'
                    '1374583608/firefox-25.0a1.en-US.linux-i686.txt'},
+    # -a firefox -t tinderbox -p win32 --extension=txt
+    {'args': {'application': 'firefox',
+              'extension': 'txt',
+              'platform': 'win32'},
+     'target': 'mozilla-central-firefox-39.0a1.en-US.win32.txt',
```

Comment:
As I have mentioned in my last review please stick with version 25.0a1! I don't want that we get too much fragmentation in our tests. We should only add new builds and versions, if something changed on ftp.mozilla.org.